src/styles/blocks/buttonBuy.scss

```diff
@@ -0,0 +1,21 @@
+.buttonBuy {
+  color: #fff;
+  background-color: $main-button-color;
+  text-decoration: none;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  box-sizing: border-box;
+  width: 166px;
+  height: 40px;
+  border-radius: 5px;
+  font-family: $main-font-famaly;
```

Comment:
There is a typo in the variable name '$main-font-famaly'. It should be '$main-font-family'. Ensure that the variable is correctly defined and imported.